octoprint_discordremote/discord.py

```diff
@@ -285,9 +285,17 @@ def handle_dispatch(self, js):
             for upload in data['attachments']:
                 filename = upload['filename']
                 url = upload['url']
-                snapshots, embeds = self.command.download_file(filename, url, user)
+
+                self.command.download_file(filename, url, user)
+                #check if file is eligible for unzipping
+                ready_to_unpack, embeds = self.command.judge_is_unzippable(filename)
```

Comment:
the self.send(embeds=embeds) will send the response before actually unzipping. There is still a possiblity that the zip will fail to unzip, which will be a bit confusing for the user.

Reply:
The response of judge_is_unzippable just tells the user it's trying to unzip ("File Received, unzipping..."), the unzip function will then report success or failure itself after it's been called. Responses are:
- single zip, auto-unzipping enabled: 'File Received, unzipping...'
- single zip, auto-unzipping disabled: 'File Received'
- multi-volume, end unknown: 'x of ??? Files Received'
- multi-volume, end known: 'x of y Files Received'
- multi-volume, all volumes received, auto-unzipping enabled: ''y of y Files Received''
- multi-volume, all volumes received, auto-unzipping disabled: 'All Files Received, unzipping...'
- file isn't a zip file: 'File Received'

Reply:
I've changed the text from "unzipping..." to "starting to unzip...", commit 3e6ecf7